### PR TITLE
Fix leave_submit

### PR DESCRIPTION
**Pull request by takidog**
v2
leave submit .

### CODE REVIEW

**Comment by vn7n24fzkq** [comment]
Thanks for your pull request.
I think it can be merged.
@abc873693 
Need your review.

**Review by abc873693 — APPROVED** [review]
Good!
Need to create document or sample request.

**Review by vn7n24fzkq — COMMENTED** [review]
For line 143 to 145 and line 157.
Do we need those comment?

**Comment by abc873693** [comment]
But I think this feature maybe to implement in v3?

File: src/kuas_api/views/v2/leave.py [hunk 1]
```diff
@@ -155,10 +153,11 @@ def leave_submit():
         end_date = "/".join(end_date)
 
         # Fixing, don't send it
```
Comment (mlouielu) [inline src/kuas_api/views/v2/leave.py:155]: As @vn7n24fzkq said, this three line can be remove (155~157)

File: src/kuas_api/views/v2/leave.py [hunk 2]
```diff
@@ -142,9 +140,9 @@ def leave_submit():
             'reason_text'] if 'reason_text' in request.form else None
         section = json.loads(
             request.form['section']) if 'section' in request.form else None
-
-        s = requests.session()
-        set_cookies(s, session['c'])
+        #s = requests.session()
```
Comment (mlouielu) [inline src/kuas_api/views/v2/leave.py:143]: Remove unused comment

**Comment by abc873693** [comment]
Because this feature will implement in v3.
Must close it first.

**Pull request closed without merging**